lib/libsnmp/asynrole.py

```diff
@@ -4,62 +4,62 @@
 #    libsnmp - a Python SNMP library
 #    Copyright (c) 2003 Justin Warren <daedalus@eigenmagic.com>
 
-import sys
 import asyncore
-import types
+import sys
 
-from libsnmp import rfc1155
-from libsnmp import rfc1157
 from libsnmp import role
 
-class manager(asyncore.dispatcher):
-    
-    def __init__(self, (cb_fun, cb_ctx), dst=(None, 0), interface=('0.0.0.0', 0), timeout=0.25):
-        
+
+class Manager(asyncore.dispatcher):
+
+    def __init__(self, xxx_todo_changeme, dst=(None, 0), interface=('0.0.0.0', 0), timeout=0.25):
```

Comment:
Can we pick a name for the variable rather than leaving it as `xxx_todo_changeme`?
Seems small and easy to achieve, since we're here.